jupyter/pytorch+llmcompressor/test/test_notebook.ipynb

```
@@ -0,0 +1,80 @@
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fc39f0e9",
+   "metadata": {
+    "vscode": {
+     "languageId": "plaintext"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "import os\n",
```

Ruff (0.12.2)

Accessing only the first or last element of `str.split()` without setting `maxsplit=1`

(PLC0207)